go/client/cmd_login.go

```diff
@@ -79,7 +79,7 @@ func (c *CmdLogin) Run() error {
 
 	err = client.Login(ctx,
 		keybase1.LoginArg{
```

Comment:
talked to max about this a bit. he brought up the fact that we cna't change the rpc since its been released so i'm going to roll back the changes to make this a union/null/string thing